iambic/plugins/v0_1_0/generic_git_provider/aws_lambda_handler.py

```diff
@@ -0,0 +1,184 @@
+from __future__ import annotations
```

Comment:
nit / non-blocking: Does this need to be an IAMbic plugin? If so, can we try to make it a standalone plugin so we can provide a reference architecture? The mono-repo approach and the `v0_1_0` in path is some existing tech debt that we'll need to work our way around sooner or later.

Reply:
probably at least 2 day investment.